azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -166,9 +167,12 @@ def start_extension_command(self, extension_name, command, cwd, env, stdout, std
                 def track_cgroups():
                     cgroups = self._cgroups_api.get_extension_cgroups(extension_name)
 
-                    # TODO: Add to tracking list
+                    for cgroup in cgroups:
```

Comment:
there is a TODO in remove_extension_cgroups that probably it is not needed anymore given the way you are pruning the tracking list (I like that)... could you remove the TODO?